source/game_sa/CarCtrl.cpp

```diff
@@ -662,13 +662,15 @@ void CCarCtrl::RemoveDistantCars() {
             if (!vehicle->vehicleFlags.bCreateRoadBlockPeds)
                 continue;
 
-            CVector centreOfWorld = FindPlayerCentreOfWorld();
+            CVector centreOfWorld = FindPlayerCentreOfWorld(CWorld::PlayerInFocus);
```

Comment:
it's default param (playerId = -1)
```suggestion
            CVector centreOfWorld = FindPlayerCentreOfWorld();
```

Reply:
It's only ever assigned 0
![obraz](https://user-images.githubusercontent.com/4185198/185786967-192ff291-ac30-483f-a61b-8b2e85ad3f28.png)

Reply:
Yeah, but if you look into how `FindPlayerCentreOfWorld` works, passing in `-1` will result in the same as passing in `CWorld::PlayerInFocus`